### PR TITLE
Deumdify

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

**Comment by gkatsev** [comment]
Modules that want to use the global `pkcs7.unpad` would need to be updated to use the `pkcs7.unpad.global.js` file.

File: dist/pkcs7.upad.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+!function(e){var f;'undefined'!=typeof window?f=window:'undefined'!=typeof self&&(f=self),(f.pkcs7||(f.pkcs7={})).unpad=e()}(function(){var define,module,exports;return (function e(t,n,r){function s(o,u){if(!n[o]){if(!t[o]){var a=typeof require=="function"&&require;if(!u&&a)return a(o,!0);if(i)return i(o,!0);var f=new Error("Cannot find module '"+o+"'");throw f.code="MODULE_NOT_FOUND",f}var l=n[o]={exports:{}};t[o][0].call(l.exports,function(e){var n=t[o][1][e];return s(n?n:e)},l,l.exports,e,t,n,r)}return n[o].exports}var i=typeof require=="function"&&require;for(var o=0;o<r.length;o++)s(r[o]);return s})({1:[function(require,module,exports){
```
Comment (dmlap) [inline dist/pkcs7.upad.js]: Where did this file come from?

File: Gruntfile.js [hunk 2]
```diff
@@ -52,7 +52,18 @@ module.exports = function (grunt) {
           'dist/pkcs7.unpad.js': 'lib/unpad.js'
         },
         options: {
-          bundleOptions: {
+          browserifyOptions: {
+            standalone: 'pkcs7.unpad'
+          }
+        }
+      },
+      upad_global: {
```
Comment (dmlap) [inline Gruntfile.js]: Spelling

File: dist/pkcs7.upad.js (hunk 1 above)
Comment (gkatsev) [inline dist/pkcs7.upad.js]: Oh, oops. I thought I deleted this one.

**Comment by dmlap** [comment]
I tried this out locally and it doesn't appear that the global is being registered.

**Pull request closed without merging**